scripts/Packages.scala

```diff
@@ -6,18 +6,53 @@
 //> using dep org.virtuslab::scala-yaml:0.0.8
 //> using file common.scala
 
+import org.virtuslab.yaml.*
+
 import java.util.Date
+import scala.concurrent.duration.Duration
+import scala.util.*
 import scala.util.control.NonFatal
-import org.virtuslab.yaml.*
 
 @main def run(args: String*): Unit =
   val packagesDir = os.pwd / ".out" / "packages"
 
+  val pluginDownloadProblemPackages = Vector(
+    "azure-native-v1",
+    "aws-s3-replicated-bucket",
+    "aws-miniflux",
+    "iosxe:0.0.1",
+    "aws-quickstart-aurora-postgres",
+    "gcp-global-cloudrun",
+    "packet",
+    "kubernetes-coredns",
+    "aws-quickstart-vpc",
+    "azure-quickstart-acr-geo-replication",
+    "aws-quickstart-redshift",
+    "nxos"
+  )
+
+  val codegenProblemPackages = Vector(
+    "aws-iam", // aws-iam:index:Account: property name "id" is reserved
+    "digitalocean", // digitalocean:index/reservedIp:ReservedIp: property name "urn"
+    // kubernetes:helm.sh/v3:Chart: property name "urn" is reserved
+    // Exception in thread "main" besom.codegen.TypeMapperError: 
+    // Failed to map type: 'NamedType(/kubernetes/v4.4.0/schema.json#/resources/kubernetes:core%2Fv1:ConfigMap,None)', asArgsType: false
+    "eks",
+    "iosxe", // schemaOnlyProvider does not implement runtime operation InitLogging
+    // error: error binding resource kubernetes:helm.sh/v3:Chart: failed to bind properties for kubernetes:helm.sh/v3:Chart: property name "urn" is reserved
+    // Exception in thread "main" besom.codegen.GeneralCodegenException: 
+    // Failed to download schema 'pulumi --logtostderr package get-schema kubernetes@4.5.5' into '.out/schemas/kubernetes/4.5.5/schema.json'
+    "kubernetes",
```

Comment:
😱

Reply:
next release should fix this:
https://github.com/pulumi/pulumi-kubernetes/commit/b045235e4f8a6c239edcf44fb1150b36c36d6c48